content/docs/user-guide/components/reference/atom/debug-rendering.md

```diff
@@ -0,0 +1,83 @@
+
+| Property | Description |
+|-|-|
+| **Debug Light Azimuth** | The azimuth in degreees used to determine the position of the debug directional light (changing this rotates the light around the up vector). |
+| **Debug Light Elevation** | Determines the elevation in degrees used to determine the position of the debug directional light (change this to move the directional light up/down). |
+| **Debug Light Color** | The color of the debug directional light. |
+| **Debug Light Intensity** | The intensity of the debug directional light. |
+
+Screenshot using the **Debug Light**
+
+{{< image-width "/images/user-guide/components/reference/atom/debug-rendering/debug-light.jpg" "1500" "Screenshot using the Debug Light" >}}
+
+
+## Material Override Properties
+
+The Debug Render Component allows users to override material values for all materials in the scene. This gives users extra options which which to vizualize and debug lighting and materials in the scene. For example, one use case would be to override the color of all objects in the scene to white or grey in order to get a better sense of the lighting in the scene.
+
+Values that can be overriden are Base Color, Roughness and Metallic. Normals maps and detail normal maps cannot be overriden but they can be disabled (if both are disabled then materials in the scene will use vertex normals only).
+
+| Property | Description |
+|-|-|
+| **Override Base Color** | Whether to override the base color for all materials in the scene to the value specified below. |
+| **Base Color Value** | The value used to override base color on all materials in the scene. For example you can set this to red to see what your scene would look like if it were entirely red. |
+| **Override Roughness** | Whether to override the roughness for all materials in the scene to the value specified below. |
+| **Roughness Value** | The value used to override roughness on all materials in the scene. For example you can set this to zero if you want to visualize a very glossy and reflective scene. |
+| **Override Metallic** | Whether to override the metallic for all materials in the scene to the value specified below. |
+| **Metallic Value** | The value used to override metallic on all materials in the scene. For example you can set this one to make all materials in your scene metals. |
+| **Enable Normal Maps** | Whether to enable/disable normal maps on all materials in the scene (note that disabling normal maps will also disable detail normal maps, and thus only the vertex normal will be used in shading calculations). |
+| **Enable Detail Normal Maps** | Whether to enable/disable detail normal maps on all materials in the scene (note that not all materials provide or support detail normal maps, therefore those materials will be unaffected by this option). |
+
+Screenshot overriding **Base Color** to **Grey (R:128, G:128, B:128)**
```

Comment:
Move to **Examples** section at the bottom of the page (per earlier comment on reorganizing).
```suggestion
Example of debug materials, with **Override Base Color** set to grey (R: `128`, G: `128`, B: `128`):
```